File: go/src/HelloWorld.go
package main

import (
  "fmt"
  "net/http"
)

const (
  port = ":24003"
)

func HelloWorld(w http.ResponseWriter, r *http.Request) {
  fmt.Fprintf(w, "HelloWorld from go!!!")
}

func init() {
  fmt.Printf("Started server")
  http.HandleFunc("/hello", HelloWorld)
  http.ListenAndServe(port, nil)
}

func main() {}

File: nodejs/HelloWorld.js
const express = require('express')
const app = express()
const port = 24001

app.get('/hello', (request, response) => {
  response.send('hello world from nodejs!!')
})

app.listen(port, (err) => {
  if (err) {
    return console.log('Doomed!!', err)
  }

  console.log(`Server is listening on ${port}`)
})
